src/rapidfuzz/process_cpp_impl.pyx

```diff
@@ -1945,3 +1999,52 @@ def cdist(queries, choices, *, scorer=ratio, processor=None, score_cutoff=None,
                 dtype, workers, &kwargs_context.kwargs)
 
     return cdist_py(queries, choices, scorer, processor, score_cutoff, score_multiplier, dtype, workers, scorer_kwargs)
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+cdef cpdist_py(queries, choices, scorer, processor, score_cutoff, score_multiplier, dtype, workers, dict scorer_kwargs):
+    # todo this should handle two similar sequences more efficiently
```

Comment:
I don't think we should care about this all to much, since at least in any sensible case this would return a list of perfect scores. In this case the best performance will be achieved by just not calling the library :man_shrugging: